ui/app/store/actions.js

```diff
@@ -372,6 +372,15 @@ var actions = {
   LOADING_TOKEN_PARAMS_STARTED: 'LOADING_TOKEN_PARAMS_STARTED',
   loadingTokenParamsFinished,
   LOADING_TOKEN_PARAMS_FINISHED: 'LOADING_TOKEN_PARAMS_FINISHED',
+
+  setSeedPhraseBackedUp,
+  showSeedPhraseBackupAfterOnboarding,
+  SHOW_SEED_PHRASE_BACKUP_AFTER_ONBOARDING: 'SHOW_SEED_PHRASE_BACKUP_AFTER_ONBOARDING',
+  hideSeedPhraseBackupAfterOnboarding,
+  HIDE_SEED_PHRASE_BACKUP_AFTER_ONBOARDING: 'HIDE_SEED_PHRASE_BACKUP_AFTER_ONBOARDING',
+
+  verifySeedPhrase,
```

Comment:
Probably better to move this function somewhere else, since it's not really an action. It feels wrong to export it from here.

Then again, it'd be a lot of work to move it without creating an additional background connection. We'd probably want to create a module for all of the background interactions. Maybe that can be a future change.